src/Components/Directory.Build.props

```diff
@@ -1,11 +1,14 @@
 <Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.props))\Directory.Build.props" />
+
   <PropertyGroup>
     <!-- Workaround for https://github.com/aspnet/AspNetCore/issues/5486 which requires the bin and obj directory be in the project directory -->
-    <OutputInRepoRoot>false</OutputInRepoRoot>
+    <BaseIntermediateOutputPath />
+    <IntermediateOutputPath />
+    <BaseOutputPath />
+    <OutputPath />
```

Comment:
Minor but why don't `$(BaseIntermediateOutputPath)` and `$(IntermediateOutputPath)` need to be cleared elsewhere?

Reply:
Blazor build tools and targets are the only thing which expect the obj/ directory to exist in the project dir. See  https://github.com/aspnet/AspNetCore/issues/5486. Everywhere else I tested this change, the projects and tests were okay with obj being moved to $repoRoot$/artifacts/obj/$projectName$